cclunit-framework-source/src/test/ccl/ut_cclut_mock.inc

```diff
@@ -226,13 +226,17 @@ end ;test_cclutRemoveAllMockImplementations_happy
 ;**********************************************************************************************************************************
 ;** cclutExecuteProgramWithMocks
 ;**********************************************************************************************************************************
-/* test_cclutExecuteProgramWithMocks_happy ******************************************************************
-*  Scenario: Executes a program using mock tables and implementations that have been set up                 *
-************************************************************************************************************/
+/* test_cclutExecuteProgramWithMocks_happy *******************************************************************************
+*  Scenario: Demonstrates a program is successfully executed using mock tables and implementations that have been set up *
+*************************************************************************************************************************/
 subroutine test_cclutExecuteProgramWithMocks_happy(null)
     declare public_subroutine = i4 with protect, noconstant(0)
     declare mock_subroutine = i4 with protect, noconstant(0)
     declare test_subroutine = i4 with protect, noconstant(0)
+    declare public::internalVariable = i4 with protect, noconstant(0)
```

Comment:
Are there any examples where the SUT declares the reply with persistscript (which is another way this "internalRecord" business could be handled)?

Reply:
I don't have any specifically using "with persistscript", no.  I can do that if you don't want me to move the declarations into the script per this question:
https://github.com/cerner/cclunit-framework/pull/7#discussion_r253161485

Reply:
I think it (persistscript) would be a good thing to cover with the tests.  I think it would be convenient to use this specific "reply" structure for that purpose.